scripts/esbuild/cli.ts

```diff
@@ -0,0 +1,75 @@
+import type { BuildOptions as ESBuildOptions } from 'esbuild';
+import * as esbuild from 'esbuild';
+import fs from 'fs-extra';
+import { join } from 'path';
+
+import { BuildOptions } from '../utils/options';
+import { writePkgJson } from '../utils/write-pkg-json';
+import { getEsbuildAliases, getEsbuildExternalModules } from './util';
+
+/**
+ * Runs esbuild to bundle the `cli` submodule
+ *
+ * @param opts build options needed to generate the rollup configuration
+ * @returns an array containing the generated rollup options
```

Comment:
Doesn't look like this actually returns anything

Reply:
🙃 right you are! I copied this JSDoc from the `cli` function for Rollup